specs/NotifyDropTargetAction.md

```diff
@@ -120,9 +122,51 @@ HRESULT DropTarget::Drop(IDataObject* dataObject,
 }
 ```
 
+## WinRT
+```c#
+// WinRT Sample
+private void WebView_DragEnter(object sender, DragEventArgs e)
+{
+  uint keyboardState = 
+    ConvertDragDropModifiersToWin32KeyboardState(e.Modifiers);
```

Comment:
@david-risney I have a concern here where the DragDropModifiers, which is what e.Modifiers returns, does not correspond easily to the win32 MK_* keys (e.g. MK_SHIFT, MK_ALT, etc..) and they have to convert it. Should we change the winrt method so that our composition controller DragEnter method directly takes in DragDropModifiers and we do the conversion?

Reply:
Ideally your winrt API works for both win32/HWND apps and UWP/CoreWindow apps (esp since PR targets both from winrt). Options:
1. Provide conversion function in sample code. Not ideal but if the conversion code is simple this is OK.
2. Provide conversion function as utility method in our API. If the conversion code isn't simple. But its a little weird for us to provide this.
3. Provide alternate versions of our methods that take DragDropModifiers. Like uint DragEnterWithDragDropModifiers(..., DragDropModifiers modifiers, ...)

Maybe (2)? 

Between MK_* & DragDropModifiers is the conversion easier in one direction than the other? Is one a superset of the other or do they both represent the same information?

Reply:
The conversion is simple.. They are one to one just with different values. (Internally, MK_* modifiers map to the VK_* modifiers but each MK value maps to a DragDropModifier and vice versa). Does this influence your answer at all?

Reply:
Then maybe (1)? Can you include the conversion method in the sample code then?

Reply:
I added a conversion. Can you review it?

Reply:
Looks good but can you add a comment somewhere in the doc for API reviewers to discuss this? 

We want a unified API surface between COM and WinRT that works in both UWP and Win32.
1. We could have one API focusing on Win32 types and require the end dev to convert from UWP types to Win32 (which we've done). 
2. Or we could have one API focusing on UWP types and require the end dev to convert Win32 to UWP. 
3. Or we could have two sets of methods one for Win32 and one for UWP types.
4. Or we could do (1) or (2) and provide a conversion function.

Because the conversion is simple and we have a large Win32 user base we chose (1).